Add "support us" section (#26)
LondonGophers/admin: sponsorship/README.md

Because we are a non-profit organisation, we will typically put any surplus funds towards conference scholarship funds such as https://www.gophercon.co.uk/scholarships/.

## 👨‍👩‍👦‍👦 Support us

> If your business is about products or services that could improve the overall meetup experience, and benefit our community, please consider supporting this group by providing us community licenses or access to free merchandise! Great examples of companies that have already agreed on this kind of "sponsorship" are:

Reviewer: ```suggestion
If your business is about products or services that could improve the overall meetup experience, and benefit our community, please consider supporting this group by providing us community licenses or access to free merchandise! Great examples of companies which have supported us this way are:
```

> * Jetbrains: three free licenses every month for our attendees to give away in a raffle fashion

Reviewer: ```suggestion
* Jetbrains: three free licenses every month for our attendees to give away in a raffle
```

> * 1Password: they granted us a community account that we can use to share admin/passwords/details within us organisers

Reviewer: ```suggestion
* 1Password: a free community license for the organisers to be able to share passwords and sensitive data securely
```

> * Stickermule: they kindly provide stickers for our events

Reviewer: ```suggestion
* Stickermule: stickers with the London Gophers logo to be distributed at our events
```

> We're incredibly grateful to everyone who would like to contribute, one way or another, and we'll make sure our friends will know about your kindness!

Reviewer: I don't think this paragraph is necessary, or adds much to what has already been said? Just trying to make sure this page doesn't get too long... :)

Author: Good point, we want to keep it short and sweet, happy to remove this last paragraph 👍

---

> As mentioned above, the organising committee of London Gophers are volunteers. We are always on the lookout for people keen to join our team!

Reviewer: Shouldn't this go with the previous section?

Author: I was thinking of having:
- Sponsorship
- Support
-> Closing

This bit doesn't look specific to either one, and putting it above sounds like it's related to sponsorship _and then_ we talk about supporters... that was a conscious decision, but if it's not clear it should be revisited 🤔
